src/shared/constants/index.ts

```diff
@@ -109,7 +109,7 @@ export const CLOUD_USAGE_PATH = '/usage'
 export const CLOUD_LOGOUT_PATH = '/logout'
 
 export const FLUX_RESPONSE_BYTES_LIMIT = CLOUD
-  ? 27 * 1024 * 1024 // 27 MiB  (desa: this was determined by looking at queries responses in the cloud app)
```

Comment:
This context might be important to share with product, that something like 99% of queries done at that time fell below this previous threshold. Maybe they can find a new "best" number?